2-mandatory/6-writing-tests-advanced.js

```diff
@@ -55,6 +55,16 @@ function formatCourseworkResult(trainee) {
     score: 63
   }
 */
+function convertScoreToGrade(score) {
```

Comment:
Think you might have a bit of stray code copied from above? You can delete this and up till the beginning of line 61 to fix